include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -713,10 +748,13 @@ class FEPointEvaluation
    */
   SmartPointer<NonMatching::MappingInfo<dim, spacedim>> mapping_info;
 
+  unsigned int current_cell_index  = numbers::invalid_unsigned_int;
+  unsigned int current_face_number = numbers::invalid_unsigned_int;
```

Comment:
I think we do not assign a value to member variables per the coding conventions. Please do it in the constructor instead. While making the change, please add a one-line comment for doxygen.